### ISSUE TITLE
dataset links

### ISSUE THREAD

**Issue by jd-coderepos** [aistairc/DeepEventMine#14]
Thank you for releasing this software with the community. 👍 💯  A small concern: would it be possible that the BioNLP dataset links are now broken?

**Comment by yanyk13** [comment]
I got the same problem. I can't open the home page of BioNLP-ST 2013...

**Comment by jd-coderepos** [comment]
It could be then that the old datasets are retired...

**Comment by yanyk13** [comment]
Any ideas where to get these datasets?

**Comment by jd-coderepos** [comment]
Not really. If they are retired perhaps then they are not publicly available anymore. But this is just my assumption.

**Comment by HevOHel** [comment]
I've tried Genia-2011 from official website of BioNLP, but there is still something wrong which is showed as file missing.

**Comment by jd-coderepos** [comment]
I have attempted to survey here the working links https://www.orkg.org/orkg/comparison/R165702 for BioNLP NER and https://www.orkg.org/orkg/comparison/R165866 for BioNLP Event Extraction.

**Comment by trieuhl** [comment]
I think you may need to contact the workshop's organizers. I found this, but not sure the data is same as the original or not:
https://sites.google.com/site/bionlpst2013/tasks/cancer-genetics-cg-task

**Comment by yanyk13** [comment]
> 

I got 404 error when opening the data links...

**Comment by yanyk13** [comment]
> https://www.orkg.org/orkg/comparison/R165866

There are no BioNLP-2013 data download links on this page

**Comment by longth-vbd** [comment]
I am sorry to hear that. Maybe you need to contact the workshop organizer:
https://aclanthology.org/W13-2008.pdf

**Comment by trieuhl** [comment]
I am sorry to hear that. Maybe you need to contact the workshop organizer:
https://aclanthology.org/W13-2008.pdf

**Comment by yanyk13** [comment]
I wrote an email to Dr.Kim two months ago, he said the website works well for him. I wrote another email with a detailed description of the data links... However, he did not reply. Got no idea where to download the data...

**Comment by yanyk13** [comment]
I wrote an email to Dr.Kim two months ago, he said the website works well for him. I wrote another email with a detailed description of the data links... However, he did not reply. Got no idea where to download the data...

**Comment by jd-coderepos** [comment]
> I think you may need to contact the workshop's organizers. I found this, but not sure the data is same as the original or not: https://sites.google.com/site/bionlpst2013/tasks/cancer-genetics-cg-task

Indeed I see a 404 error too for the data links on the page.

**Comment by jd-coderepos** [comment]
Here is a link I found on PubMed https://www.ncbi.nlm.nih.gov/research/bionlp/Data/ . Sharing in case it may be helpful to someone.

**Comment by OStars** [comment]
Do you get the GE2013 now? Are there some alternative datasets for biomedical event extraction?

**Comment by jd-coderepos** [comment]
Some listed here 'Surveying BioNLP Shared Tasks Corpora for Event Extraction' https://orkg.org/comparison/R165866/ . For those without links, maybe the authors need to be contacted directly. If you get it could you please let us know... :)

**Comment by OStars** [comment]
I find GE2013 dataset which is the same as GE2016. GE2016 reuse the GE2013, which is re-complemented and re-designed for evaluation from KB construction perspective. For details, you could read the paper: [Extending the evaluation of Genia Event task toward knowledge base construction and comparison to Gene Regulation Ontology task](https://doi.org/10.1186/1471-2105-16-S10-S3)

**Comment by OStars** [comment]
> I find GE2013 dataset which is the same as GE2016. GE2016 reuse the GE2013, which is re-complemented and re-designed for evaluation from KB construction perspective. For details, you could read the paper: [Extending the evaluation of Genia Event task toward knowledge base construction and comparison to Gene Regulation Ontology task](https://doi.org/10.1186/1471-2105-16-S10-S3)

GE2016 is now available at: http://pubannotation.org/collections/bionlp-st-ge-2016

**Comment by yanyk13** [comment]
> 

cool, thx a lot

**Comment by barthfab** [comment]
Not sure if this is still relevant but there is a git repo containing all bionlp datasets used by DeepEventMine ([openbiocorpora](https://github.com/orgs/openbiocorpora/repositories?type=all)) except for GE2016. Also, the [bigbio](https://github.com/bigscience-workshop/biomedical) library provides biomedical dataloaders for over 126+ biomedical datasets.
